configuration/file_test.go

```diff
@@ -126,7 +126,18 @@ func TestFileStoreConfiguration(t *testing.T) {
 	conf.InitDefaults()
 	d = &file{}
 
-	err := d.storeConfiguration(conf)
+	// create temp file where we can read the conf
+	tmpf, err := ioutil.TempFile("", "configuration.yaml")
+	if err != nil {
+		panic(err)
+	}
+	defer syscall.Unlink(tmpf.Name())
```

Comment:
I've used Unlink as it's the one used in other tests, but os.Remove has more documentation and I guess it's easier to catch, so will send other PR to change the other ones as well, thanks.